src/design/plone/ioprenoto/adapters/stringinterp.py

```diff
@@ -62,3 +63,26 @@ def safe_call(self):
             booking_folder.getPhysicalPath()[len(portal.getPhysicalPath()) :]  # noqa
         )
         return f"{portal_url}/{booking_folder_path}?tab=search&SearchableText={self.context.getBookingCode()}&login=1"
+
+
+@adapter(Interface)
+class BookingUnitaOrganizzativaTitle(BaseSubstitution):
+    def safe_call(self):
+        # The PrenotazioniFolder c.t. object which contains the contextual booking
+        prenotazioni_folder = self.context.getPrenotazioniFolder()
+
+        # The Relation objects to related offices of PrenotazioniFolder
+        uffici_correlati_relations = prenotazioni_folder.uffici_correlati
```

Comment:
Non puoi tornare "" se non una relation? ti eviti inutili calcoli/giri nelle righe dopo.
Idem sotto. Ha senso fare delle getattr se sai già che non hai l'oggetto che ti serve?

cc\ @mamico

Reply:
https://google.github.io/eng-practices/review/reviewer/standard.html

Reply:
It is just the code readability choice